### PR TITLE
Flesh out Results view UI

### PR DESCRIPTION
**Pull request by jtquach1**
This PR provides the basic structure of the results view including listing resulting studies. Much of the data is still hard coded and notated with `TODO` comments. This also uses mock results data which will be hooked up at a later time.

### CODE REVIEW

**Comment by noranda** [comment]
@jtquach1 and @dmpotter44: This PR has been updated and is ready for review.

**Comment by jtquach1** [comment]
The results view looks squashed on xs screens; e.g. Pixel XL (although arguably one could just rotate the phone to get the sm screen size). 
image

Maybe the study status box ("closed to accrual") could become 12-columns wide on xs screens?

Besides that, I think everything looks good to me. I'll flesh out `getDistance` and `getClosestFacility` in #10 .

**Comment by noranda** [comment]
> The results view looks squashed on xs screens

Fixed the responsive layout of the sidebar to help with this. Take a look and let me know what you think!

File: src/components/Results/StudyHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            )}
+          </Stack>
+        </Stack>
+
+        <IconButton aria-label="save study">
+          <SaveIcon fontSize="medium" sx={{ color: 'common.blue' }} />
+        </IconButton>
+      </Stack>
+    </AccordionSummary>
+  );
```
[the remark this answers is not in the document]
Comment (noranda) [reply src/components/Results/StudyHeader.tsx]: Pushed up a commit to resolve this. Decided to keep the status as a small tag on the top left when at very small widths. Let me know what you think!

Screen Shot 2021-11-18 at 1 45 52 PM

**Comment by jtquach1** [comment]
This looks good to me! I didn't know about AccordionActions, that's really nifty.

**Comment by jtquach1** [comment]
> This looks good to me! I didn't know about AccordionActions, that's really nifty.

My bad, I missed this earlier- on xs screens (iPhone 5 size) there's a bit of clipping: 
image

I think that can be resolved with changing the base font size for xs screens, but also the menu hamburger icon/touch area is cut off on screens <400px wide.
image

I can take a stab at these two issues since I have time for the end of this week.

**Comment by jtquach1** [comment]
# Screenshots of current visual changes:
| xs screen (Pixel 2)  | sm screen (Surface Duo) | md screen (iPad) | lg screen (iPad Pro) | xl screen (MacBook Pro) | Sidebar on xs screen (Galaxy Fold, Pixel 2) | Sidebar on non-xs screen (Surface Duo, iPad) |
| - | - | - | - | - | - | - |
| image  | image | image | image | image | image | image |
| image  | image  | image | image | image | image | image |

**Review by noranda — CHANGES_REQUESTED** [review]
This is looking very good! Nice job on improving the responsiveness. The only thing I think we need to fix is the following console warning: `Warning: Received `false` for a non-boolean attribute `shrink`.` when the results first render (tested on iPhone X). Otherwise all lint and tests pass and app works as expected.

**Comment by jtquach1** [comment]
> This is looking very good! Nice job on improving the responsiveness. The only thing I think we need to fix is the following console warning: `` Warning: Received `false` for a non-boolean attribute `shrink`. `` when the results first render (tested on iPhone X). Otherwise all lint and tests pass and app works as expected.

Thanks for catching that! The error seems to have disappeared upon checking `shrink` as one of the props to not forward to the DOM (latest commit).

**Pull request merged**